### PR TITLE
Wei du

### PR DESCRIPTION
**Pull request by chef27615**

### CODE REVIEW

File: ANSWERS.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+- PropTypes will check and ensoure the Props passing between the parent and child componenets will be in the right type and shape. It will allow developer to check the console and know right props was passed and avoid future bugs in the program.
```
Comment (jheinz1999) [inline ANSWERS.md:3]: Excellent answer!

File: ANSWERS.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Describe a life-cycle event in React?
+
+- example: 
+    ComponentDidMount will link the data from off site server to local program, and trigger a re-render of the component
```
Comment (jheinz1999) [inline ANSWERS.md:8]: Good answer, can you tell me when componentDidMount will run?

File: ANSWERS.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Explain the details of a Higher Order Component?
+
+- HOC is a function take in parameter(s) and return a component.
```
Comment (jheinz1999) [inline ANSWERS.md:12]: Just note that the parameters to higher order components are other components

File: ANSWERS.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+What are three different ways to style components in React? Explain some of the benefits of each.
```
Comment (jheinz1999) [inline ANSWERS.md:14]: Perfect answers!

File: lambdatimes/package.json [hunk 5]
```diff
@@ -3,11 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "prop-types": "^15.6.2",
+    "prop-types": "^15.7.2",
```
Comment (jheinz1999) [inline lambdatimes/package.json:6]: Great work pulling in the right modules!

File: lambdatimes/src/components/Content/Card.js [hunk 6]
```diff
@@ -1,19 +1,30 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Card = props => {
```
Comment (jheinz1999) [inline lambdatimes/src/components/Content/Card.js:4]: Great work on this component!

File: lambdatimes/src/components/Content/Cards.js [hunk 7]
```diff
@@ -1,15 +1,31 @@
...
+        card={card}
+        key={index}
+      />)}
+
     </div>
   )
 }
 
 // Make sure you include prop types for all of your incoming props
+Cards.propTypes= PropTypes.shape({
```
Comment (jheinz1999) [inline lambdatimes/src/components/Content/Cards.js]: Note that propTypes should be an object here, with the key being the name of the prop and the value being the PropTypes.shape()

File: lambdatimes/src/components/Content/Content.js [hunk 8]
```diff
@@ -6,23 +6,34 @@ import Cards from './Cards';
...
+      selected: '',
       tabs: [],
       cards: []
     };
   }
 
   componentDidMount() {
+    
     // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState({
```
Comment (jheinz1999) [inline lambdatimes/src/components/Content/Content.js:24]: Great work!

File: lambdatimes/src/components/Content/Tab.js [hunk 9]
```diff
@@ -1,13 +1,23 @@
...
+    //   return props.tab.className="tab active-tab"
+    // } else{
+    //   return props.tab
+    // }
+
   return (
     <div
-      className={''}
-      onClick={() => {
+      className={ props.tab === props.selectedTab ? "tab active-tab" : props.tab}
```
Comment (jheinz1999) [inline lambdatimes/src/components/Content/Tab.js]: Note that if the tab is not selected, the class name should be "tab"

File: lambdatimes/src/components/Content/Tab.js [hunk 10]
```diff
@@ -1,13 +1,23 @@
...
+      onClick={(tab) => { props.selectedTabHandler(tab)
```
Comment (jheinz1999) [inline lambdatimes/src/components/Content/Tab.js]: Take note of the names of the props that you are passing in

File: lambdatimes/src/components/Header.js [hunk 11]
```diff
@@ -1,15 +1,41 @@
 import React from 'react';
-
+import styled from 'styled-components';
 // Refactor this component to use styled components and not classNames. 
 // You can find the corresponding CSS in the CSS/index.css file
 
+const HeaderStyle= styled.header`
```
Comment (jheinz1999) [inline lambdatimes/src/components/Header.js:6]: Awesome work with styled components!

**Review by jheinz1999 — CHANGES_REQUESTED** [review]
Awesome work Wei! Just a few minor changes need to be made to hit MVP. You're doing great!

File: ANSWERS.md (hunk 2 above)
Comment (chef27615) [reply ANSWERS.md:8]: after the first render

**Pull request merged**